scripts/publish-ppa.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#exit when any command fails
+set -e
+cd /home/runner/work/sedge/sedge/sedge
+mkdir -p build/package/debian/src/github.com/NethermindEth/sedge/
+rsync -aq . build/package/debian/src/github.com/NethermindEth/sedge/ --exclude build/ --exclude .git/ --exclude docs/ --exclude scripts/
```

Comment:
Let's remove scripts from this exclude. I believe it caused issues with the pattern on templates.go file.

Reply:
Fixed